pymc_experimental/statespace/core/statespace.py

```diff
@@ -69,6 +66,16 @@ def _verify_group(group):
         raise ValueError(f'Argument "group" must be one of "prior" or "posterior", found {group}')
 
 
+def _matrix_vector_dot(A: pt.TensorVariable, x: pt.TensorVariable) -> pt.TensorVariable:
```

Comment:
Is there a way to do this in "pure pytensor", without writing a helper?

Reply:
`(A @ x[..., None]).squeeze(-1)`?